src/skin/tooltips.cpp

```diff
@@ -869,4 +869,13 @@ void Tooltips::addStandardTooltips() {
     add("skin_settings")
             << tr("Skin Settings Menu")
             << tr("Show/hide skin settings menu");
+
+    // Sampler Bank Controls
```

Comment:
The tooltips are short. If you feel they should be more in detail, e.g. that technical not the samples are saved, please speak up.